packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/TextBoundsProvider.java

```diff
@@ -40,6 +46,13 @@ public class TextBoundsProvider {
 
     private static String fontName;
 
+    static {
+        // Frontend does not render the font the same way than the backend (it appears bigger on the frontend).
```

Comment:
I don't have any issue with this code or its comment but there was a time where this did work as expected.

The fact that it does not seems to work is not the expected state. Something has probably been changed maybe on the frontend and we should probably fix it properly instead of applying some hot fix here. I'll open an issue to remind us to investigate that and fix it properly.

Reply:
https://github.com/eclipse-sirius/sirius-components/issues/1647